pkg/lvm/lvm_util.go

```diff
@@ -59,6 +60,34 @@ const (
 	YES = "yes"
 )
 
+// LogicalVolume specifies attributes of a given lv that exists on the node.
+type LogicalVolume struct {
+
+	// Name of the lvm logical volume
+	Name string `json:"name"`
+
+	// Full name of the lvm logical volume
```

Comment:
Can you give an example of the Name and FullName in the comments, so that we dont get confused between them.